modules/Sources/Features/Scan/ScanStore.swift

```diff
@@ -38,7 +38,8 @@ public struct Scan {
         public init(
             info: String = "",
             isTorchAvailable: Bool = false,
-            isTorchOn: Bool = false
+            isTorchOn: Bool = false,
+            isCameraEnabled: Bool = true
```

Comment:
This change does not seem to be related to this PR. Is it intention?

Reply:
It's a fix for unit tests, the init method was lacking option to set a different value.